svir/dialogs/drive_oq_engine_server_dialog.py

```diff
@@ -631,23 +634,29 @@ def show_output_list(self, output_list, calculation_mode):
                 self.connect_button_to_action(button, action, output, outtype)
                 self.output_list_tbl.setCellWidget(row, col, button)
                 self.calc_list_tbl.setColumnWidth(col, BUTTON_WIDTH)
-            if output['type'] in (OQ_ALL_LOADABLE_TYPES |
-                                  OQ_RST_TYPES |
-                                  OQ_NO_MAP_TYPES):
-                if output['type'] in OQ_RST_TYPES | OQ_NO_MAP_TYPES:
-                    action = 'Show'
-                else:
-                    action = 'Load as layer'
-                # TODO: remove check when gmf_data will be loadable also for
-                #       event_based
-                if (output['type'] == 'gmf_data'
-                        and calculation_mode == 'event_based'):
-                    continue
-                button = QPushButton()
-                self.connect_button_to_action(
-                    button, action, output, outtype)
```

Comment:
I changed the indentation here, although it worked also the other way. I think it's logically clearer now.